<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>GraphSettings2dDialog</class>
 <widget class="QDialog" name="GraphSettings2dDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>400</width>
    <height>338</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="minimumSize">
   <size>
    <width>400</width>
    <height>320</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>400</width>
    <height>338</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Ustawienia okna wykresu 2d</string>
  </property>
  <property name="styleSheet">
   <string notr="true">ColorButton {text-align:left; padding-left:10px}</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <layout class="QFormLayout" name="formLayout_2">
     <property name="fieldGrowthPolicy">
      <enum>QFormLayout::AllNonFixedFieldsGrow</enum>
     </property>
     <item row="0" column="0">
      <widget class="QCheckBox" name="showAxes">
       <property name="text">
        <string>Pokaż osie układu współrzędnych</string>
       </property>
      </widget>
     </item>
     <item row="0" column="1">
      <widget class="ColorButton" name="axesColorBtn">
       <property name="minimumSize">
        <size>
         <width>0</width>
         <height>23</height>
        </size>
       </property>
       <property name="layoutDirection">
        <enum>Qt::LeftToRight</enum>
       </property>
       <property name="autoFillBackground">
        <bool>false</bool>
       </property>
       <property name="styleSheet">
        <string notr="true"/>
       </property>
       <property name="text">
        <string>Kolor osi</string>
       </property>
       <property name="flat">
        <bool>false</bool>
       </property>
      </widget>
     </item>
     <item row="1" column="0">
      <widget class="QCheckBox" name="showLabels">
       <property name="text">
        <string>Pokaż etykiety</string>
       </property>
      </widget>
     </item>
     <item row="1" column="1">
      <widget class="ColorButton" name="labelsColorBtn">
       <property name="minimumSize">
        <size>
         <width>0</width>
         <height>23</height>
        </size>
       </property>
       <property name="text">
        <string>Kolor etykiet</string>
       </property>
      </widget>
     </item>
     <item row="2" column="0">
      <widget class="QCheckBox" name="showGrid">
       <property name="text">
        <string>Pokaż siatkę</string>
       </property>
      </widget>
     </item>
     <item row="2" column="1">
      <widget class="ColorButton" name="gridColorBtn">
       <property name="minimumSize">
        <size>
         <width>0</width>
         <height>23</height>
        </size>
       </property>
       <property name="text">
        <string>Kolor siatki</string>
       </property>
      </widget>
     </item>
     <item row="3" column="1">
      <widget class="ColorButton" name="bgColorBtn">
       <property name="minimumSize">
        <size>
         <width>0</width>
         <height>23</height>
        </size>
       </property>
       <property name="text">
        <string>Kolor tła</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <widget class="QGroupBox" name="drawInInterval">
     <property name="title">
      <string>Kreśl w przedziale</string>
     </property>
     <property name="checkable">
      <bool>true</bool>
     </property>
     <property name="checked">
      <bool>false</bool>
     </property>
     <layout class="QGridLayout" name="gridLayout">
      <item row="0" column="0">
       <widget class="QLabel" name="label">
        <property name="text">
         <string>Min. X:</string>
        </property>
       </widget>
      </item>
      <item row="0" column="1">
       <widget class="QLineEdit" name="intervalXFrom"/>
      </item>
      <item row="0" column="2">
       <widget class="QLabel" name="label_2">
        <property name="text">
         <string>Max. X:</string>
        </property>
       </widget>
      </item>
      <item row="0" column="3">
       <widget class="QLineEdit" name="intervalXTo"/>
      </item>
      <item row="1" column="1">
       <widget class="QLineEdit" name="intervalYFrom"/>
      </item>
      <item row="1" column="2" rowspan="2">
       <widget class="QLabel" name="label_4">
        <property name="text">
         <string>Max. Y:</string>
        </property>
       </widget>
      </item>
      <item row="1" column="3" rowspan="2">
       <widget class="QLineEdit" name="intervalYTo"/>
      </item>
      <item row="1" column="0">
       <widget class="QLabel" name="label_3">
        <property name="text">
         <string>Min. Y:</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="QCheckBox" name="showLegend">
     <property name="text">
      <string>Pokaż legendę</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QCheckBox" name="enableAntialiasing">
     <property name="text">
      <string>Włącz antialiasing</string>
     </property>
    </widget>
   </item>
   <item>
    <spacer name="verticalSpacer">
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>20</width>
       <height>40</height>
      </size>
     </property>
    </spacer>
   </item>
   <item>
    <widget class="QDialogButtonBox" name="buttonBox">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
     <property name="standardButtons">
      <set>QDialogButtonBox::Cancel|QDialogButtonBox::Ok</set>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>ColorButton</class>
   <extends>QPushButton</extends>
   <header>colorbutton.h</header>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections>
  <connection>
   <sender>buttonBox</sender>
   <signal>rejected()</signal>
   <receiver>GraphSettings2dDialog</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>325</x>
     <y>310</y>
    </hint>
    <hint type="destinationlabel">
     <x>286</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>buttonBox</sender>
   <signal>accepted()</signal>
   <receiver>GraphSettings2dDialog</receiver>
   <slot>checkSettings()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>270</x>
     <y>293</y>
    </hint>
    <hint type="destinationlabel">
     <x>373</x>
     <y>268</y>
    </hint>
   </hints>
  </connection>
 </connections>
 <slots>
  <slot>checkSettings()</slot>
 </slots>
</ui>
